timepiece/reports/views.py

```diff
@@ -2183,3 +2183,30 @@ def get_context_data(self, **kwargs):
 
         context['pending_milestones'] = pending_milestones
         return context
+
+#TODO
+class ThroughputReport(TemplateView):
+    template_name = 'timepiece/reports/throughput.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(ThroughputReport, self).get_context_data(**kwargs)
+        projects = []
+        for project in Project.objects.filter(status__label='Archived'):
```

Comment:
Why is this hard-coded to Archived?  Also, change this to be a setting that references an ID.